syntax.php

```diff
@@ -10,7 +10,6 @@
 if(!defined('DOKU_INC')) die();
 
 if(!defined('DOKU_PLUGIN')) define('DOKU_PLUGIN',DOKU_INC.'lib/plugins/');
-require_once(DOKU_PLUGIN.'syntax.php');
```

Comment:
The two lines above this line and the comment can be deleted as well.

Reply:
Aha ok thanks. :-)  I was just looking at what lines of code were throwing warnings within Dokuwiki.